src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -18,6 +18,7 @@ function AnnotationsTargetedTooltip({ children, isEnabled = false }: Props): JSX
             className="bp-AnnotationsTooltip"
             shouldTarget={isEnabled}
             showCloseButton
+            tetherElementClassName="bp-AnnotationsTooltip-element"
```

Comment:
This class name is a little confusing because this element appears "above" `bp-AnnotationsTooltip` in the DOM and hyphens are used to indicate descendancy in SUIT CSS naming conventions. Could we use `bp-AnnotationsTooltipTether`?